io/private/EptSupport.hpp

```diff
@@ -271,29 +263,29 @@ class PDAL_DLL EptInfo
         Binary
     };
 
-    EptInfo(Json::Value info) : m_info(info)
+    EptInfo(const NL::json& info) : m_info(info)
     {
         m_bounds = toBox3d(m_info["bounds"]);
-        m_points = m_info["points"].asUInt64();
-        m_span = m_info["span"].asUInt64();
-        m_srs = m_info["srs"]["wkt"].asString();
+        m_points = m_info["points"].get<uint64_t>();
+        m_span = m_info["span"].get<uint64_t>();
+        m_srs = m_info["srs"]["wkt"].get<std::string>();
```

Comment:
This is UB if `srs` does not have a `wkt` member, which is allowed by EPT if there is no associated SRS.  See first point of [further notes](https://github.com/nlohmann/json#further-notes).

Recommend something like
```
if (m_info.contains("srs"))
    m_srs = m_info.at("srs").value("wkt", "");
```